Osnova.Net/Constants/DtfSubsiteIds.cs

```diff
@@ -0,0 +1,482 @@
+﻿namespace Osnova.Net.Constants
+{
```

Comment:
There's some overall improvements:

- Add an explicit `public` modifier to the class please
- Would be great to add link in xml-docs of class where to find subsite list (for the future updates)
- Separate properties with one line, e.g:

```c#
/// <summary>
/// https://dtf.ru/games
/// </summary>
public const int Games = 64953;

/// <summary>
/// https://dtf.ru/gameindustry
/// </summary>
public const int GameIndustry = 87855;
```

As additional point it would be awesome to sort all ids from lowest value to highest to easify search/modification/adding of them in code.

Also, take a look at my addition to `public const int Video`, I defined some kind of standard for comments on these classes.

Thanks for your contribution!